OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -1016,20 +1016,27 @@ void TracerThread::ProcessDeferredEvents() {
   pthread_setname_np(pthread_self(), "Proc.Def.Events");
   bool should_exit = false;
   while (!should_exit) {
+    ORBIT_SCOPE("ProcessDeferredEvents iteration");
     // When "should_exit" becomes true, we know that we have stopped generating
     // deferred events. The last iteration will consume all remaining events.
     should_exit = stop_deferred_thread_;
     std::vector<std::unique_ptr<PerfEvent>> events = ConsumeDeferredEvents();
     if (events.empty()) {
       // TODO: use a wait/notify mechanism instead of check/sleep.
+      ORBIT_SCOPE("Sleep");
       usleep(IDLE_TIME_ON_EMPTY_DEFERRED_EVENTS_US);
     } else {
-      for (auto& event : events) {
-        int fd = event->GetOriginFileDescriptor();
-        event_processor_.AddEvent(fd, std::move(event));
+      {
```

Comment:
The extra braces are not needed here, if I'm not mistaken. I would remove them (but also see other comment about consistency).

Reply:
Without these braces this would overlap with `ORBIT_SCOPE("ProcessOldEvents");`.

Reply:
Ah, I mis-matched the braces, thought this was the only part in the else. Disregard then.